tests/devapps/DesktopTestApp/MainForm.cs

```diff
@@ -637,6 +636,36 @@ private async void B2cSilentCustomDomain_Click(object sender, EventArgs e)
             }
         }
 
+        private void RopcB2CAT_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                _publicClientHandler.PublicClientApplication = PublicClientApplicationBuilder
+                   .Create(_b2CClientId)
+                   .WithB2CAuthority(B2CROPCAuthority)
+                   .BuildConcrete();
+                GetUserDataForUserNamePasswordFlow();
+            }
+            catch (Exception exc)
+            {
+                CreateException(exc);
+            }
+        }
+
+        private async void GetUserDataForUserNamePasswordFlow()
```

Comment:
The name of the method indicates that the function is getting the data for the User Name Password Flow. However, the return type is void and the method makes the acquiretoken as well. I would suggest combining the functionality of both the RopcB2CAT_Click and the AcquireTokenByUsernamePasswordAsync into one and changing the signature to accept no parameters since the user data will always be acquired from the UI. Other acquiretoken methods in the class follow this pattern.